### PR TITLE
[Doc] Fix Icons doc

### PR DESCRIPTION
**Pull request by JorickPepin**
| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | no 
| Issues        | 
| License       | MIT

This sentence appears twice

### CODE REVIEW

File: src/Icons/doc/index.rst [hunk 1]
```diff
@@ -167,9 +167,6 @@ a subdirectory, the *name* will be ``subdirectory:icon_name``.
 Icons On-Demand
 ~~~~~~~~~~~~~~~
 
-`ux.symfony.com/icons`_ has a huge searchable repository of icons
```
Comment (WebMamba) [inline src/Icons/doc/index.rst:170]: @smnandre is it the first one or the second one we should keep? the on-demande is different

Comment (smnandre) [reply src/Icons/doc/index.rst:170]: I think in RST it's  `_`  to emphasis

Comment (JorickPepin) [reply src/Icons/doc/index.rst:170]: the first does not render correctly

https://symfony.com/bundles/ux-icons/current/index.html#icons-on-demand

image

Comment (smnandre) [reply src/Icons/doc/index.rst:170]: Well thne it's `**like that**` 😅

Comment (javiereguiluz) [reply src/Icons/doc/index.rst:170]: Yes, the `_ ... _` is italics in Markdown. In RST, italics is `* ... *` and bold is `** ... **`

But here the problem is that this phrase is wrongly repeated, so let's remove it entirely.

**Comment by javiereguiluz** [comment]
Thank you Jorick.

**Pull request merged**